collection-scripts/gather_etcd

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+BASE_COLLECTION_PATH="/must-gather"
+ETCD_LOG_PATH="${BASE_COLLECTION_PATH}/etcd_info"
+
+ETCD_CONTAINER='etcdctl'
```

Comment:
can we clarify this ENV as `ETCDCTL_CONTAINER` so that I don't have a mini heart attack every time I see this code?

Reply:
Ok. No problem